gradle.properties

```
@@ -9,7 +9,7 @@ yarn_mappings=1.20.1+build.2
 loader_version=0.14.21
 
 # Mod Properties
-mod_version=1.4.0
+mod_version=1.4.1
```

```suggestion
mod_version=1.5.0
```
should be minor version instead of patch